ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayFeatureLayers/DisplayFeatureLayers.qml

```diff
@@ -0,0 +1,215 @@
+// [WriteFile Name=DisplayFeatureLayers, Category=Layers]
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import Esri.ArcGISRuntime
+import Esri.ArcGISExtras
+
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property url dataPath: {
+        Qt.platform.os === "ios" ?
+                    System.writableLocationUrl(System.StandardPathsDocumentsLocation) + "/ArcGIS/Runtime/Data/" :
+                    System.writableLocationUrl(System.StandardPathsHomeLocation) + "/ArcGIS/Runtime/Data/"
+    }
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Component.onCompleted: {
+            // Set and keep the focus on MapView to enable keyboard navigation
+            forceActiveFocus();
+        }
+
+        Map {
+            id: map
+            initBasemapStyle: Enums.BasemapStyleArcGISTopographic
+        }
+    }
+
+    Rectangle {
+        id: layerSelectRectangle
+        anchors {
+            top: parent.top
+            left: parent.left
+            margins: 5
+        }
+        width: 225
+        height: column.height + 10
+        border.color: "black"
+
+        ColumnLayout {
+            id: column
+            anchors.centerIn: parent
+            Text {
+                text: "Feature Layer Mode"
+            }
+            ComboBox {
+                id: featureLayerComboBox
+                implicitWidth: layerSelectRectangle.width - 10
+                model: ["Geodatabase", "GeoPackage", "Portal Item", "Service Feature Table", "Shapefile"]
+                onCurrentTextChanged: {
```

Comment:
the C++ version is setup to work onCurrentIndexChanged. They are similar, but I've seen differences in the past. For example, on instantiation, currentTextChanged will likely fire, whereas currentIndexChanged will not. It's worth testing and make sure behavior is consistent